src/store_cache.rs

```diff
@@ -73,19 +70,21 @@ impl StoreCache {
 
             Ok(data)
 
-        }, |data| {
-            Ok(data)
-        })
+        },
+                           |data| Ok(data))
```

Comment:
This one is especially weird. Looks like it tried to align with the `(` from `map_or_else`. Though `map_or_else` felt weird to use anyway, so I wouldn't mind if this was rewritten.